### PR TITLE
Updates from the jaeger maintainers

### PR DESCRIPTION
**Pull request by jkowall**
I have updated the text from us at Jaeger, please let me know if you have questions.

ref : https://github.com/cncf/tag-app-delivery/issues/734

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]

**Review by danieloh30 — APPROVED** [review]
LGTM

**Comment by jkowall** [comment]
Sorry I deleted my fork. You can merge this in after if you want but didn't get any approvals.

**Comment by danieloh30** [comment]
I think @ThomasVitale can do it

**Pull request closed without merging**
